qatrack/qatrack_core/forms.py

```
@@ -15,13 +15,67 @@ def label_from_instance(self, obj):
+            form=self.form,
+            name=name,
+            boundfields=boundfields,
+            legend=options.get('legend', name.replace('_', ' ').title() if name else None),
+            classes=classes_opt,
+            description=options.get('description', ''),
+        )
+        self.fieldsets.append(fieldset)
+        if name:
+            self._dict[name] = fieldset
+
+    def __iter__(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__iter__`

(ANN204)